unit test failures on macOS with nix

I'm getting this unit test failure when building the project on a new macOS machine with nix:

```
         /nix/store/6pk5ri23bbgpmx18kksh5kni5sr01vas-python3.13-weasyprint-68.1-unstable-2026-05-18
       Last 25 log lines:
       >     def assert_pixels_equal(name, width, height, raw, expected_raw, tolerance=0):
       >         """Take 2 matrices of pixels and assert that they are the same."""
       >         if raw != expected_raw:  # pragma: no cover
       >             pixels = zip_longest(raw, expected_raw, fillvalue=(-1, -1, -1))
       >             for i, (value, expected) in enumerate(pixels):
       >                 if expected is None:
       >                     continue
       >                 if any(abs(value - expected) > tolerance
       >                        for value, expected in zip(value, expected)):
       >                     actual_height = len(raw) // width
       >                     write_png(name, raw, width, actual_height)
       >                     expected_raw = [
       >                         pixel or (255, 255, 255) for pixel in expected_raw]
       >                     write_png(f'{name}.expected', expected_raw, width, height)
       >                     x = i % width
       >                     y = i // width
       > >                   pytest.fail(
       >                         f'Pixel ({x}, {y}) in {name}: '
       >                         f'expected rgba{expected}, got rgba{value}')
       > E                   Failed: Pixel (7, 1) in unicode_range: expected rgba(255, 255, 255), got rgba(255, 0, 0)
       >
       > tests/draw/__init__.py:110: Failed
       > =========================== short test summary info ============================
       > FAILED tests/draw/test_text.py::test_unicode_range - Failed: Pixel (7, 1) in unicode_range: expected rgba(255, 255, 255), got rg...
       > ==== 1 failed, 4017 passed, 41 deselected, 39 xfailed in 143.69s (0:02:23) =====

```

--- a/tests/draw/test_text.py
+++ b/tests/draw/test_text.py
@@ -1310,9 +1310,9 @@ def test_unicode_range(assert_pixels):
         }
         body {
           color: red;
-          font-family: uni;
+          font-family: uni, %s;
           font-size: 2px;
           line-height: 0;
           margin: 2px 1px;
         }
-      </style>ADZB''')
+      </style>ADZB''' % SANS_FONTS)

